src/cf-pagination/src/molecules/pagination.less

```diff
@@ -0,0 +1,107 @@
+//
+// Theme variables
+//
+
+// Color variables
+// $color- variables are from 18F's US Web Design Standards
+// https://github.com/18F/web-design-standards/blob/18f-pages-staging/src/stylesheets/core/_variables.scss
+
+@pagination-text-color: #75787B;
```

Comment:
Any idea what this color is? It doesn't seem to be listed there. The closest is `$color-gray-medium:          #757575; // lightest gray that passes color contrast`